Visualization/generate_BackBLAST_heatmap.R

```diff
@@ -0,0 +1,686 @@
+#!/usr/bin/env Rscript
```

Comment:
We might want to reformat this to follow the tidyverse r style guide. https://style.tidyverse.org Its by the guy who wrote GG plot.

Reply:
All functions are now defined roxygen style